app/controllers/api/guides_controller.rb

```diff
@@ -33,5 +38,26 @@ def destroy
                                           user: current_user)
       respond_with_mutation(:no_content)
     end
+
+    private
+
+    def check_if_crop_exists(params)
```

Comment:
Assignment Branch Condition size for check_if_crop_exists is too high. [19.75/15]